packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+.rt-DataListLabel {
+  color: var(--gray-a11);
+  box-sizing: border-box;
+  display: block;
+  white-space: nowrap;
```

Comment:
Don't think we need `white-space: nowrap;`?

Reply:
This was on the original component and since we don't set any default widths the labels get scrunched without it, but I'm fine to remove and folks can add it. 

<img width="351" alt="Screenshot 2024-01-22 at 9 04 29 AM" src="https://github.com/radix-ui/themes/assets/36613477/4ed5876f-724b-4af2-982a-79d709d7539a">